source/compiler/aslmap.c

```diff
@@ -339,6 +339,7 @@ const ASL_MAPPING_ENTRY     AslKeywordMapping [] =
 /* FINDSETRIGHTBIT */           OP_TABLE_ENTRY (AML_FIND_SET_RIGHT_BIT_OP,  0,                              0,                  ACPI_BTYPE_INTEGER),
 /* FIXEDDMA */                  OP_TABLE_ENTRY (AML_DEFAULT_ARG_OP,         0,                              0,                  0),
 /* FIXEDIO */                   OP_TABLE_ENTRY (AML_DEFAULT_ARG_OP,         0,                              0,                  0),
+/* DWORDPCC */                  OP_TABLE_ENTRY (AML_DEFAULT_ARG_OP,         0,                              0,                  0),
```

Comment:
It looks like entries in this file are alphabetical- so DWORDPCC would go between DWORDMEMORY and DWORDSPACE.
This seems to be true for other places where code is inserted in this PR also.

Reply:
Fixed